__tests__/index.test.js

```diff
@@ -0,0 +1,30 @@
+const app = require('../app.js');
+const request = require('supertest');
+const db = require('../db/connection.js');
+const seed = require('../db/seeds/seed.js');
+const {articleData,commentData,topicData,userData}= require('../db/data/test-data/index.js');
+
+beforeEach(()=>seed(topicData,userData,articleData,commentData));
+afterAll(()=>db.end());
+
+describe('/api/topics',()=>{
+    test('GET 200 status coded return on get request',()=>{
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+    })
+
+    test('GET request should return an array of topic objects',()=>{
+        return request(app)
+        .get('/api/topics')
+        .then(({body})=>{
+            expect(body.topics).toHaveLength(3);
+            body.topics.forEach((topic)=>{
+                expect(typeof topic.slug).toBe('string');
+                expect(typeof topic.description).toBe('string');
+            })
+        })
+    })
+
+
+})
```

Comment:
Really nice testing on this endpoint using forEach and checking the type of the values. You can also use this syntax -'expect(topic).toHaveProperty('slug', expect.any(String))'. Nice to see that you've remembered to check the length of the body also.

Don't forget to test for 404 errors on any endpoint that does not exist eg '/api/banana'